scenarios/centos-9/podified_common.yml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
The comment talks about EDPM multinode, but then cifmw_deploy_edpm: false. If this is a common file for EDPM and non-EDPM deployment maybe the comment should reflect it

Reply:
Thank you @tosky .Good point, I will update that in the main pr.